gems/bundler/CVE-2016-7954.yml

```diff
@@ -11,6 +11,11 @@ description: |
   code into an application by leveraging a Gem name collision on a
   secondary source.
 
+  '*****************************************************************
+  Please note that this vulnerability only applies for Ruby projects
+  using Bundler < 2.0 with Gemfiles having 2 or more "source" lines.
+  *****************************************************************'
```

Comment:
Maybe also explicitly mention that if the user's `Gemfile` does not use multiple sources, the vulnerability can be ignored.

Reply:
Added text